public/apps/configuration/constants.tsx

```diff
@@ -138,12 +138,18 @@ export const INDEX_PERMISSIONS: string[] = [
   'indices:admin/cache/clear',
   'indices:admin/close',
   'indices:admin/create',
+  'indices:admin/data_stream/create',
+  'indices:admin/data_stream/delete',
+  'indices:admin/data_stream/get',
   'indices:admin/delete',
   'indices:admin/exists',
   'indices:admin/flush',
   'indices:admin/flush*',
   'indices:admin/forcemerge',
   'indices:admin/get',
+  'indices:admin/index_template/delete',
```

Comment:
any other index_template actions?

Reply:
I have checked thoroughly Opensearch code to get exhaustive list of operations supported w.r.t. index templates. I have found only 3 actions which I have included them here.

Reply:
`indices:admin/index_template/simulate` and `indices:admin/index_template/simulate_index`?

Reply:
Seems some issue with my editor, rechecked again. These 2 actions specified by you complete the list of all supported actions for index templates. Adding them in this list.